### PR TITLE
update with 4.2.3 screenshots and fix formatting error

### PR DESCRIPTION
**Pull request by alexarobu**

### CODE REVIEW

**Comment by archaeogeek** [comment]
@alexarobu this now needs adjusting to take into account recent changes to creating-custom-editor.rst before it can be merged

**Comment by alexarobu** [comment]
@archaeogeek one of the old commits slipped in; I removed it now, all should be good

File: source/user-guide/describing-information/img/inspire-thesaurus-selection.png
Comment (archaeogeek) [inline source/user-guide/describing-information/img/inspire-thesaurus-selection.png:1]: This one includes Scotgov specific thesauri that should be removed

**Review by archaeogeek — COMMENTED** [review]
There's one image that includes scotgov specific thesauri but otherwise it looks good

File: source/user-guide/describing-information/img/inspire-thesaurus-selection.png
Comment (alexarobu) [reply source/user-guide/describing-information/img/inspire-thesaurus-selection.png:1]: should be good now I think @archaeogeek

**Comment by alexarobu** [comment]
@archaeogeek I think this is all good now :smile:

**Review by archaeogeek — APPROVED** [review]

**Pull request merged**